src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,151 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.exception.InvalidInputDataException;
+import core.basesyntax.service.exception.UserAlreadyExistsException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "validLogin";
+    private static final String VALID_PASSWORD = "validPassword123";
+    private static final Integer VALID_AGE = 33;
+    private static final Integer NEGATIVE_AGE = -1;
+    private static final Integer AGE_LESS_THAN_18 = 17;
+    private static final String SHORT_LOGIN = "usr";
+    private static final String SHORT_PASSWORD = "pwd";
+    private static final String EXISTING_LOGIN = "existingUser";
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_validUser_ok() {
+        User user = new User();
+        user.setLogin(VALID_LOGIN);
+        user.setPassword(VALID_PASSWORD);
+        user.setAge(VALID_AGE);
+
+        User registeredUser = registrationService.register(user);
+
+        assertNotNull(registeredUser, "Registered user should not be null");
+        assertEquals(VALID_LOGIN, registeredUser.getLogin());
+        assertEquals(VALID_PASSWORD, registeredUser.getPassword());
+        assertEquals(VALID_AGE, registeredUser.getAge());
+    }
```

Comment:
The test 'register_validUser_ok' is good, but ensure you have a test case for when a user with a valid age of exactly 18 is registered, as this is an edge case.